wifiphisher/pywifiphisher.py

```diff
@@ -925,6 +925,9 @@ def run():
 
     time.sleep(3)
 
+    #no longer need mac_matcher
+    mac_matcher.unbind()
```

Comment:
Another approach is to set `mac_to_vendor` to a empty dictionary like so:
```python
self.mac_to_vendor = {}
```
This way it will not crash and just returns `unknown`